src/store/modules/GlobalStore.js

```diff
@@ -49,10 +49,18 @@ export const GlobalStore = defineStore('global', {
   }),
   getters: {
     bootProgressGetter: (state) => state.bootProgress,
+    isInPhypStandby: (state) =>
+      // SystemHardwareInitializationComplete and after is "PHYP in standby"
+      state.bootProgress === 'SystemHardwareInitializationComplete' ||
+      state.bootProgress === 'SetupEntered' ||
+      state.bootProgress === 'OSBootStarted' ||
+      state.bootProgress === 'OSRunning',
     isOSRunningGetter: (state) => state.bootProgress === 'OSRunning',
     getIsUtcDisplay: (state) => state.isUtcDisplay,
     safeModeGetter: (state) => state.safeMode,
     serverStatusGetter: (state) => state.serverStatus,
+    isServiceUser: (state) =>
+      state.currentUser?.RoleId === 'OemIBMServiceAgent' || !state.currentUser,
```

Comment:
state currentUser is neither initialised nor assigned a value anywhere in the file. getCurrentUser() and getAccountService() functions need to be included from older file. Or directly state userPrivilege can be used as well to get the role id of the user.